app/models/manageiq/providers/hawkular/datawarehouse_manager/event_catcher/runner.rb

```diff
@@ -0,0 +1,105 @@
+class ManageIQ::Providers::Hawkular::DatawarehouseManager::EventCatcher::Runner <
```

Comment:
I think using the mixin in the hawkular provider event catcher is feasible.  After it has been running well in the DW provider we can schedule a task to re-use it in hwk provider.